build/webpack/webpack.client.config.js

```diff
@@ -35,7 +35,8 @@ module.exports = {
             memoryLimit: 9096
         }),
         new DefinePlugin({
-            scriptUrl: 'import.meta.url'
+            scriptUrl: 'import.meta.url',
+            'process.env': '{}' // utils references `process.env.xxx`
```

Comment:
Do we happen to know more specifically what utils needs? What if something else is also looking at process.env?

Reply:
>   What if something else is also looking at process.env?

That's why its empty. if something tries to use it, the variable will not be defined.
In webworker we don't have access to `process`.

> Do we happen to know more specifically what utils needs? 

Yes, documented further below where `util` is used (util.promisify).